src/libServer/LookupServer.cpp

```diff
@@ -558,6 +558,12 @@ Json::Value LookupServer::CreateTransaction(
 
     Transaction tx = JSONConversion::convertJsontoTx(_json);
 
+    if (tx.IsEth()) {
+      throw JsonRpcException(RPC_INVALID_PARAMETER,
+                             "Eth txs not supported for CreateTransaction api "
+                             "- use eth_sendRawTransaction");
+    }
+
```

Comment:
Yes! It is time! 

We had used this interface for EVM as well initially until we had eth_sendRawTransaction.